.github/workflows/ci.yml

```diff
@@ -17,6 +17,16 @@ env:
   RUSTFLAGS: "-Dwarnings"
 
 jobs:
+  build:
```

Comment:
Just to document the fix

Reply:
I don't think it is necessary. The bug seems to be in `wasm-bindgen`, so we shouldn't add any regressions for it.